install_magenta.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Check which operating system
+if [ "$(uname)" == "Darwin" ]; then
+    echo "Mac OS Detected"
+    export OS="MAC"
+    export MINICONDA_SCRIPT=Miniconda2-latest-MacOSX-x86_64.sh
+    # Mac OS X, CPU only, Python 2.7:
+    export TF_BINARY_URL=https://storage.googleapis.com/tensorflow/mac/cpu/tensorflow-0.11.0rc0-py2-none-any.whl
+elif [ "$(uname)" == "Linux" ]; then
+    echo "Linux OS Detected"
+    export OS="LINUX"
+    export MINICONDA_SCRIPT=Miniconda2-latest-Linux-x86_64.sh
+    # Ubuntu/Linux 64-bit, CPU only, Python 2.7
+    export TF_BINARY_URL=https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.0rc0-cp27-none-linux_x86_64.whl
+fi
+
+
+# Check if anaconda already installed
+if [ ! $(which conda) ]; then
+    echo ""
+    echo "==========================================="
+    echo "anaconda not detected, installing miniconda"
+    echo "==========================================="
+    echo ""
+    curl https://repo.continuum.io/miniconda/$MINICONDA_SCRIPT > $/tmp/$MINICONDA_SCRIPT
+    bash $/tmp/$MINICONDA_SCRIPT
+    source $HOME/.bash_profile
+else
+    echo ""
+    echo "==================================="
+    echo "anaconda detected, skipping install"
+    echo "==================================="
+    echo ""
+fi
+
+# Set up the magenta environment
+echo ""
+echo "=============================="
+echo "setting up magenta environment"
+echo "=============================="
+echo ""
+
+conda create -n magenta python=2.7
+source activate magenta
+
+# Install tensorflow
+pip install --ignore-installed --upgrade $TF_BINARY_URL
+
+# Install other dependencies
+pip install jupyter magenta
```

Comment:
I'm not completely sure, but I think if you include jupyter on the 'conda create' command or use 'conda install' , it can use the cached jupyter package, which the user might already have if they have installed the full anaconda environment before.